tests/manage/pv_services/test_pvc_concurrent_deletion_creation.py

```diff
@@ -18,6 +18,7 @@
 
 
 @bugzilla('1734259')
+@bugzilla('1745022')
```

Comment:
I don't think we follow this way of mapping bug to test case. bugzilla marker is used for skipping test case if bz is not verified. This is to avoid unnecessary execution and failure.

@red-hat-storage/top-level-reviewers Please confirm

Reply:
checked, you are right
removing it